test/unittests/parser_test.cpp

```diff
@@ -664,3 +664,50 @@ TEST(parser, block_br)
         "01000000"
         "01000000"_bytes);
 }
+
+TEST(parser, instr_br_table)
+{
+    // (block
```

Comment:
Can you use `/* */` comment block here too?

Reply:
done